js/index.js

```diff
@@ -33,3 +33,9 @@ window.addEventListener('mouseup', function () {
     mouseIsPressed = false
 })
 createGrid()
+
+var color = document.getElementById('eraser').getAttribute('data-color')
```

Comment:
Can you change document.getElementById('eraser') to buttonClicked as the element will be assigned to that variable?

Reply:
buttonClick